### PR TITLE
[OCPQE-15225] HyperShift upgrade automation pipeline build -- Nodepool(InPlace) upgrade

### PR DESCRIPTION
**Pull request by huangmingxia**
/cc @LiangquanLi930 @heliubj18 @liangxia 

This PR is nodepool upgradeType with InPlace. Would you mind review it when you free?  Thanks.

[OCPQE-15225](https://issues.redhat.com/browse/OCPQE-15225 )

### CODE REVIEW

**Comment by huangmingxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-4.13-upgrade-from-stable-4.13-aws-ipi-ovn-hypershift-worker-node-inplace-p1-f14

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.13.yaml [hunk 1]
```diff
@@ -232,6 +232,23 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test-hypershift-worker-node
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-guest
+- as: aws-ipi-ovn-hypershift-worker-node-inplace-p1-f14
```
Comment (LiangquanLi930) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.13.yaml:235]: Could you help to change `aws-ipi-ovn-hypershift-worker-node-p1-f4` to `aws-ipi-ovn-hypershift-worker-node-replace-p1-f4`

Comment (LiangquanLi930) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.13.yaml:235]: We should change the test frequency to f7 for worker-node upgrade

**Comment by huangmingxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-4.13-upgrade-from-stable-4.13-aws-ipi-ovn-hypershift-worker-node-inplace-p1-f14

**Comment by huangmingxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-4.13-upgrade-from-stable-4.13-aws-ipi-ovn-hypershift-worker-node-inplace-p1-f7

**Comment by huangmingxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-4.13-upgrade-from-stable-4.13-aws-ipi-ovn-hypershift-worker-node-inplace-p1-f7

**Comment by huangmingxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-4.13-upgrade-from-stable-4.13-aws-ipi-ovn-hypershift-worker-node-replace-p1-f7

**Pull request merged**